### DISCUSSION TITLE
使用sysinfo crates进行系统状态的获取以获取更多平台的支持

### DISCUSSION THREAD

**Discussion by Shonke** [zdz/ServerStatus-Rust#140] [Ideas]
sysinfo已经是引入了的
是否可以用sysinfo来获取CPU内存等系统状态，这样就可以在macOS/Windows机器上使用了
我尝试过删除status部分是可以在macOS上运行的

**Comment by Shonke** [comment]
改了一部分数据的获取，是可以正常上报的
<img width="981" alt="image" src="https://user-images.githubusercontent.com/9344384/215314381-72e81123-f0af-4a03-9d4b-5ccdeeeee6a4.png">

**Comment by zdz** [comment]
欢迎测试 [v1.7.0](https://github.com/zdz/ServerStatus-Rust/releases/tag/v1.7.0)
